### PR TITLE
Enforce synchronous writing of the file on OS X

### PR DESCRIPTION
**Pull request by RomanSaveljev**
My fix for #32

### CODE REVIEW

File: mbed_flasher/flashers/FlasherMbed.py [hunk 1]
```diff
@@ -27,6 +27,7 @@
 from enhancedserial import EnhancedSerial
 from serial.serialutil import SerialException
 import hashlib
+import fcntl
```
Comment (RomanSaveljev) [inline mbed_flasher/flashers/FlasherMbed.py]: aww this is not needed

**Comment by RomanSaveljev** [comment]
@jupe @VeliMattiLahtela does it look like it could just cut it?

**Comment by jupe** [comment]
lgtm

**Comment by VeliMattiLahtela** [comment]
lgtm

**Pull request merged**